src/syscheckd/src/db/src/fimDBSpecialization.h

```diff
@@ -256,6 +265,13 @@ class FIMDBCreator<OSType::WINDOWS> final
                                     nlohmann::json::parse(FIM_REGISTRY_START_CONFIG_STATEMENT),
                                     syncRegistryMessageFunction);
         }
+
+        static void encodeString(std::string& stringToEncode)
+        {
+#ifdef WIN32
+            stringToEncode = Utils::EncodingWindowsHelper::stringAnsiToStringUTF8(stringToEncode);
+#endif
+        }
```

Comment:
I'm not to fond of leaving this macro, as we have been working to remove them in C++ code. The way to fix this should be to also parametrize  `EncodingWindowsHelper`, but this will also impact in other modules that make use of this.